DDD-11.md

```diff
@@ -0,0 +1,165 @@
+# Universal Transaction Buffer
+
+## Motivation
+
+Most Debezium connectors are provided transaction log changes for committed transactions, meaning that the connector can immediately dispatch changes safely.
+However, for some connectors like Oracle and iSeries, the transaction logs contain a mix of committed, uncommitted, and rolled back changes.
```

Comment:
```suggestion
However, for some connectors like Oracle and Informix, the transaction logs contain a mix of committed, uncommitted, and rolled back changes.
```

Also there is https://debezium.io/documentation/reference/3.1/connectors/mysql.html#mysql-property-binlog-buffer-size